meta/recipes-kernel/lttng/babeltrace2_2.0.3.bb

```diff
@@ -17,7 +17,7 @@ UPSTREAM_CHECK_GITTAGREGEX = "v(?P<pver>2(\.\d+)+)$"
 
 S = "${WORKDIR}/git"
 
-inherit autotools pkgconfig ptest python3targetconfig
+inherit autotools pkgconfig ptest
```

Comment:
This commit needs a signoff and an explanation for why we're reverting the target commit.

Reply:
Added.